clues.go

```diff
@@ -99,7 +99,7 @@ func normalize(kvs ...any) map[string]any {
 
 		var value any
 		if i+1 < len(kvs) {
-			value = kvs[i+1]
+			value = marshal(kvs[i+1])
```

Comment:
is this access and the for-loop indexing above safe? I haven't really seen any checks here or in caller code to ensure that we have an even number of entries

Reply:
This is the even-number check.